docs/topics/home.xml

```diff
@@ -37,11 +37,12 @@
                 <a href="whatsnew1920.md" description="Latest features: Kotlin K2 is in Beta and Stable Kotlin Multiplatform">What's New in Kotlin 1.9.20</a>
                 <a href="roadmap.md" description="Future plans on Kotlin development">Kotlin public roadmap</a>
             </group>
-            <group type="cards">
+            <group type="cards" display="wide">
                 <title>Kotlin Multiplatform</title>
-                <a href="multiplatform-get-started.md" description="Useful links to help you start using the SDK">Get started</a>
-                <a href="multiplatform-plugin-releases.md" description="Features of the Kotlin Multiplatform Mobile plugin">Plugin releases for Android Studio</a>
-                <a href="https://www.youtube.com/playlist?list=PLlFc5cFwUnmy_oVc9YQzjasSNoAk4hk_C" description="Videos about Kotlin Multiplatform on our YouTube channel">Kotlin Multiplatform Multiverse</a>
+                <a href="https://www.jetbrains.com/kotlin-multiplatform/" description="Learn how Kotlin Multiplatform helps you reuse code in your applications">Why Kotlin Multiplatform</a>
```

Comment:
The key word for KMP is "share": Kotlin Multiplatform helps you _share_ code between your applications.